app/views/notes/_comments.html.erb

```diff
@@ -17,8 +17,8 @@
     <% if current_user %>
       <%= render :partial => "comments/form", :locals => { title: I18n.t('notes._comments.post_comment'), comment: false, placeholder: I18n.t('notes._comments.post_placeholder') } %>
     <% else %>
-      <p><a href="#" data-toggle="modal" data-target="#loginModal">
```

Comment:
Just want to know what is the issue with `loginModal`. We used it on entire site in last GCI.
Thanks both of you. Please tell the task number so that I can make the instance count as 2.
Each will get 3 points.

Reply:
@SidharthBansal here it is: https://codein.withgoogle.com/dashboard/task-instances/5973563488600064/

Reply:
Just want to know what is the issue with loginModal. We used it on entire site in last GCI.

Reply:
@SidharthBansal I've investigated it a bit and look at this:

![Screenshot_20200112_172444](https://user-images.githubusercontent.com/35997844/72222027-7ee08f80-3560-11ea-85df-fac6c6894ae7.png)

Reply:
That's why @Uzay-G's solution works.

Reply:
That's not his question. In my login url the return_to parameter is unescaped which is essential for redirection. He is wondering why you removed the login-modal part. That is what I think he is asking about maybe

Reply:
We redirect to /login, we don't display a modal.

Reply:
Because of consistency. In the production the first button redirects to /login and the button that we've fixed opens a modal.

Reply:
Wait one second let me investigate this

Reply:
I think we shouldn't edit the button that opens the modal. We only need to change the other one that already redirected to "/login".

Reply:
I will look at this when I can and fiddle with the problem locally :+1:

Reply:
We should display login modal on both buttons instead of redirecting to /login?
Is that what you have in mind?

Reply:
No I think the button in the `_comments` file should open the modal like in production and we should maintain the fixes to the login button in `_comment`

Reply:
I'll return it then. Thanks.

Reply:
No problem :+1: